CHANGELOG.md

```diff
@@ -438,7 +438,7 @@ Full changelog @  https://github.com/drwetter/testssl.sh/commits/2.2/testssl.sh
 * local hostname in output
 
 1.31 (Halloween Release)
-* bugfix: SSLv2 was kind of borken
+* bugfix: SSLv2 was kind of broken
```

Comment:
this is deliberate ;-)

Reply:
Ah, will fix back to _borken_ then :smile: